pyproject.toml

```diff
@@ -854,7 +853,12 @@ ignore = [
     "COM819",
 
     # Disabled because ruff does not understand type of __all__ generated by a function
-    "PLE0605"
+    "PLE0605",
+
+    "TC006",
```

Comment:
should we add a comment as to why these are disabled?
why are the separated from the other rules here?